about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="./styles/main.css"> 
+  <link rel="stylesheet" type="text/css" href="./styles/normalize.main.css">
+  <title>About</title>
+</head>
+
+<!-- Navigation Station and Contact/Social Media Links  -->
+<div class="navbar">
+  <nav>
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="blog.html">Blog</a></li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+      <li><a class="active" href="about.html">About</a></li>
+      <li style="float:right"><a href="https://twitter.com/SesamesSeeds">Twitter</a></li>
+      <li style="float:right"><a href="mailto:sesameseed@rushtheweb.com">E-Mail</a></li>
+    </ul>  
+  </nav>
+</div>
+
+<!-- Background -->
+<body background="./images/cat.jpg">
```

Comment:
instead of using the `background` attribute for the background image, use CSS.  This is old-style HTML.